packages/components/pie-textarea/test/component/pie-textarea.spec.ts

```diff
@@ -407,6 +411,180 @@ test.describe('PieTextarea - Component tests', () => {
                 expect(isValid).toBe(true);
             });
         });
+
+        test.describe('maxLength', () => {
+            test('should not display a form label when the label is absent but maxLength is provided', async ({ mount, page }) => {
+                // Arrange
+                const component = await mount(PieTextarea, {
+                    props: {
+                        maxLength: 10,
+                    } as TextareaProps,
+                });
+
+                // Act
+                const labelNodes = component.locator('pie-form-label');
+
+                // Assert
+                await expect(labelNodes).toHaveCount(0);
+            });
+
```

Comment:
Note for reviewers: I initially wrote an additional test that checks that pasting in content does not exceed the max length. Unfortunately, I cannot get this test to work on CI. Only locally. If we want this test, I would prefer to work on it in a follow-up test excellence ticket as I do not feel it justifies blocking this feature.